python3.6/Dockerfile

```diff
@@ -40,30 +43,30 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # Create working directory
-ENV WORKING_DIR=/opt/invenio
-ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
-RUN mkdir -p ${INVENIO_INSTANCE_PATH}
+ENV PROJECT_DIR=/opt/${PROJECT_NAME}
```

Comment:
The renaming here to `PROJECT_` is made to align with the cookiecutter project name / package name.